scoop/src/main/java/com/lyft/scoop/Router.java

```diff
@@ -24,7 +27,9 @@ public boolean goBack() {
                 Scoop nextScoop = backStack.peek();
                 Screen nextScreen = Screen.fromScoop(nextScoop);
                 performScoopChange(nextScoop, nextScreen, previousScreen, TransitionDirection.EXIT);
-
+                return true;
+            } else if (hasEmptyStack) {
```

Comment:
No because dismiss will be called on every back pressed, so if there is nothing in the stack this method will return false.